lutris-sc-install.json

```diff
@@ -32,7 +32,13 @@
       "script": {
         "files": [
           {
-            "client": "https://install.robertsspaceindustries.com/star-citizen/RSI-Setup-1.6.8.exe"
+            "client": "https://install.robertsspaceindustries.com/star-citizen/RSI-Setup-1.6.9.exe"
+          },
+          {
+            "prelaunch": "https://gist.githubusercontent.com/GloriousEggroll/289ff1051743c6f89e4b1189f5da5ea8/raw/88ec1898f3d31df60d17125e609f9671db9cf749/sc-prelaunch.sh"
```

Comment:
I'd rather keep the prelaunch command rather than running a random script from the web.

Reply:
Agreed, this is why we've been doing it this way as a pre-launch command where the user can easily see exactly what's happening.

In theory, a fixed gist commit shouldn't change so we could inspect that GE gist once and probably trust it forever, but we don't live in a perfect world where "in theory" means "in reality". Generally, fetching scripts from the web is a Very Bad Idea(TM) so that's what we're going with.

Reply:
Fair enough, I pulled this from Lutris, so I that would explain why that is different here :)

Can adjust that fairly easily.